neuralprophet/forecaster.py

```diff
@@ -1858,22 +1874,23 @@ def plot_latest_forecast(
             fcst, self.config_train.quantiles, n_last=1 + include_previous_forecasts
         )
 
-        # Check whether the default plotting backend is overwritten
-        plotting_backend = (
-            plotting_backend
-            if plotting_backend != "default"
-            else (self.plotting_backend if hasattr(self, "plotting_backend") else "matplotlib")
-        )
+        # Check whether a local or global plotting backend is set.
+        if hasattr(self, "plotting_backend") and plotting_backend is None:
+            plotting_backend = select_plotting_backend(self.plotting_backend)
+        else:
+            plotting_backend = select_plotting_backend(plotting_backend)
```

Comment:
Minor improvement idea: we have these lines of code several time, how about extracting it into a method?